syntax = "proto3";

message CmdDigital {
    string channel = 1;
    bool value = 2;
}

message CmdDigitalMultiple {
    repeated CmdDigital values = 1;
}